pkg/crd/crd.go

```diff
@@ -98,7 +98,7 @@ func newRestClient(config *rest.Config) (r *rest.RESTClient, err error) {
 
 func newBindingWatcher(r *rest.RESTClient, eventCh chan aadpodid.EventType) (cache.SharedInformer, error) {
 	azBindingWatcher := cache.NewSharedInformer(
-		cache.NewListWatchFromClient(r, aadpodid.AzureIDBindingResource, "default", fields.Everything()),
+		cache.NewListWatchFromClient(r, aadpodid.AzureIDBindingResource, v1.NamespaceAll, fields.Everything()),
```

Comment:
Is this a breaking change?
Before we were only watching `default` now all namespaces?

Reply:
With the new changes it will look exactly the same. For unnamespaced identities. everything will be created on the default namespace